api/v1beta1/activemqartemis_types.go

```diff
@@ -741,4 +741,8 @@ const (
 	ConfigAppliedConditionUnknownReason                   = "UnableToRetrieveStatus"
 	ConfigAppliedConditionOutOfSyncReason                 = "OutOfSync"
 	ConfigAppliedConditionNoJolokiaClientsAvailableReason = "NoJolokiaClientsAvailable"
+
+	BrokerVersionAlignedConditionType           = "BrokerVersionAligned"
+	BrokerVersionAlignedConditionMatchReason    = "VersionMatch"
```

Comment:
wondering if we need a reason when the condition is true, the condition type, BrokerVersionAligned is informative when true. But I see we do have a precedent for this with other positive reason strings.

Reply:
I agree with you for "normal-true" conditions the reason is useless but the k8s status client fails when the reason is empty, i.e.:
```
... status.conditions[x].reason in body should be at least 1 chars long", Reason: "Invalid", Details: *(*"k8s.io/apimachinery/pkg/apis/meta/v1.StatusDetails")(0xc00055d0e0), Code: 422}}
```